net.lutris.Lutris.yml

```diff
@@ -56,6 +56,16 @@ add-extensions:
     no-autodownload: true
     autodelete: true
 
+  com.valvesoftware.Steam.Utility:
+    subdirectories: true
+    directory: utils
+    version: beta
+    versions: beta;test
```

Comment:
This will allow only `beta` and `test` branches of Steam extension to be attached to Lutris. Probably it's the correct behavior, since Lutris itself is in `beta`, but keep this in mind, just in case.

Reply:
But users wont probably have the stable extension?